{
  "timestamp": "2025-04-06T21:39:42.966929",
  "error_type": "TypeError",
  "error_message": "BaseService.transaction() takes 1 positional argument but 2 were given",
  "traceback": "Traceback (most recent call last):\n  File \"/home/andrii/Documents/Projects/Mathtermind/src/services/auth_service.py\", line 593, in reset_password\n    with self.transaction(self.db) as session:\n         ~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/usr/lib/python3.13/contextlib.py\", line 305, in helper\n    return _GeneratorContextManager(func, args, kwds)\n  File \"/usr/lib/python3.13/contextlib.py\", line 109, in __init__\n    self.gen = func(*args, **kwds)\n               ~~~~^^^^^^^^^^^^^^^\nTypeError: BaseService.transaction() takes 1 positional argument but 2 were given\n",
  "system_info": {
    "platform": "Linux-6.12.19-1-lts-x86_64-with-glibc2.41",
    "python_version": "3.13.2",
    "hostname": "archlinux",
    "pid": 1371339,
    "cwd": "/home/andrii/Documents/Projects/Mathtermind",
    "user": "andrii",
    "executable": "/home/andrii/Documents/Projects/Mathtermind/venv/bin/python"
  },
  "additional_data": {
    "context": {
      "token": "abcdef12"
    }
  }
}